bluebottle/projects/serializers.py

```diff
@@ -81,7 +81,7 @@ def get_attribute(self, obj):
         return obj
 
     rewards = RelatedResourcePermissionField('reward-list')
-    donations = ResourcePermissionField('order-manage-list')
```

Comment:
Does this change need a test?

Reply:
I guess it should.

I'll add one.

Reply:
I added some tests that failed before the fix and passes after